### ISSUE TITLE
Subs2Cia omitting some subtitle lines

### ISSUE THREAD

**Issue by spirited1086** [dxing97/subs2cia#31]
Just noticed that subs2cia will omit some lines when parsing an srt subtitle file, e.g., in my case there were a lot of <i> or </i> tags and those  lines were omitted when condensing. The resuting condensed audio was a mess. I removed all the <i> tags and it's working. Is this a feature? if so what other kind of characters should I search for and remove from subtitle files to allow for propery condensation?

**Comment by dxing97** [comment]
Yes, there's a built in filter for removing suspected non-dialogue subtitle lines. You can check out the relevant code [here.](https://github.com/dxing97/subs2cia/blob/master/subs2cia/subtools.py#L438) If it's not working with your files you can disable it with `-ni` and optionally use your own regex instead with `-R`.
